web/internal/ui/src/tailwind.css

```
@@ -0,0 +1,331 @@
+Raw variables are defined in :root/.dark blocks for use in inline styles (e.g. charts).
+The @theme block registers Tailwind utility classes for each color.
+Semantic variables (--background, --content, etc.) define the light/dark theme.
+
+In Tailwind v4, this file should be @import'd from your main Tailwind CSS file.
+*/
+
+/* ============================================================
+   Tailwind @theme registration — generates utility classes
+   (e.g. bg-gray-1, text-error-9, border-accent-6, etc.)
+   ============================================================ */
+@theme inline {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)